.github/workflows/workflow.yaml

```diff
@@ -301,3 +304,19 @@ jobs:
                       create_output(result["Target"], misconfiguration)
 
               sys.exit(exit_status)
+
+  trivy-sbom:
+    name: Generate SBOM report of application in ${{ inputs.working-directory }}
+    runs-on: ${{ inputs.runs-on }}
+    if: inputs.trivy-sbom-enabled
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Run Trivy in GitHub SBOM mode and submit results to Dependency Snapshots
```

Comment:
A little shorter name here would also be nice. We already know it's going to run Trivy in SBOM mode, so it could suffice with

> Generate report

Reply:
How does this end up in Dependency Snapshots as it stands now? Is there some magic with the filename, or?

Reply:
> How does this end up in Dependency Snapshots as it stands now? Is there some magic with the filename, or?

https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/exploring-the-dependencies-of-a-repository#viewing-the-dependency-graph

By checking the dependency graph path in Insights.

Reply:
> A little shorter name here would also be nice. We already know it's going to run Trivy in SBOM mode, so it could suffice with
> 
> > Generate report

Fixed.